src/shared/lib/analytics/client.tsx

```diff
@@ -4,17 +4,21 @@ import { env } from "@/env";
 
 const isProd = process.env.NODE_ENV === "production";
 
-const AnalyticsProvider = () => (
-  <OpenPanelComponent
-    clientId={env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID!}
+const AnalyticsProvider = function() {
+  if(!env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID){
+    return null;
+  }
+
+  return <OpenPanelComponent
+    clientId={env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID}
     trackAttributes={true}
     trackOutgoingLinks={isProd}
     trackScreenViews={isProd}
   />
-);
+};
 
 const track = (options: { event: string } & PostEventPayload["properties"]) => {
-  if (!env.NEXT_PUBLIC_OPENPANEL_ENABLED) {
+  if (!env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID || !isProd ) {
```

Comment:
```suggestion
  if (!env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID) {
```

Reply:
isnt this more like the other case where we prevent sending events in no prod? @Snouzy

Reply:
it's the client analytics (in opposition to the server). So maybe something like that could be more "identical" let's say?

```tsx
const track = (options: { event: string } & PostEventPayload["properties"]) => {
  // we don't want to use it
  if (!env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID) {
    return;
  }

  // here we want to use it, but let's check if we are in prod or no

  // eslint-disable-next-line react-hooks/rules-of-hooks
  const { track: openTrack } = useOpenPanel();

  const { event, ...rest } = options;

  if (!isProd) {
    console.log("Track", { event, ...rest });
    return;
  }

  openTrack(event, rest);
};
```